.eslintrc.js

```diff
@@ -0,0 +1,46 @@
+module.exports = {
+  env: { browser: true, node: true },
+  root: true,
+  parser: "@typescript-eslint/parser",
+  plugins: [
+    "@typescript-eslint",
+    "import",
+    "simple-import-sort",
+    // 'prettier' commented as we don't want to run prettier through eslint because of performance degradation
+  ],
+  extends: [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+    "prettier/@typescript-eslint", // Uses eslint-config-prettier to disable ESLint rules from @typescript-eslint/eslint-plugin that would conflict with prettier
+  ],
+  rules: {
+    /**
+     * eslint-plugin-import @see https://github.com/benmosher/eslint-plugin-import
+     */
+    "import/order": 0, // turn off in favor of eslint-plugin-simple-import-sort
+    "import/no-unresolved": 0,
+    "import/no-duplicates": 1,
+    /**
+     * eslint-plugin-simple-import-sort @see https://github.com/lydell/eslint-plugin-simple-import-sort
+     */
+    "sort-imports": 0, // we use eslint-plugin-import instead
+    "simple-import-sort/imports": 1,
+    "simple-import-sort/exports": 1,
+    /**
+     * @typescript-eslint/eslint-plugin @see https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/eslint-plugin
+     */
```

Comment:
@colinhacks please let me know whether we should follow the recommended rules below that I temporarily disabled

Reply:
Whatever you think is best here, I'm not opinionated about linting.